src/util/shared/record-display-helper.ts

```diff
@@ -23,14 +23,16 @@ export function postProcessRecord(record: any, attributes: Attribute[]) {
 
 export function postProcessAttributes(attributes: Attribute[]) {
     const prepareAttributes = jsonCopy(attributes);
-    if (!attributes[0].hasOwnProperty('key')) {
-        prepareAttributes.forEach((attribute, index) => {
-            if (attribute.id !== null) {
-                attribute.key = attribute.id;
-            } else {
-                throw new Error("Cant find attribute id in attribute object");
-            }
-        });
+    if (attributes && attributes.length > 0) {
```

Comment:
we usually use it the other way around, so we would not have to nest the code one more space
`if (not ... ) return; `

Reply:
In this specific scenario we need to also check if the array is not empty, as the first condition can still pass for empty arrays. So that check is still required as we are accessing the index directly

Reply:
yes i know, but I just meant the negation to the whole expression so we don't have to write the whole code one tab inside
the comment can be also ignored, don't mind it